.prettierrc.yml

```diff
@@ -0,0 +1,3 @@
+singleQuote: true
+semi: false
```

Comment:
We should enable semicolons here or in a follow-up. Our TypeScript eslint config demands it.

Reply:
follow up, please 😅

Reply:
Haha, okay, _fine_!